.expeditor/config.yml

```diff
@@ -37,4 +44,6 @@ subscriptions:
 
   - workload: staged_workload_released:{{agent_id}}:release_staging:*
     actions:
+      - built_in:bump_version
```

Comment:
I wonder how this is going to work with the date format? Maybe we shouldn't have expeditor auto bump and only we bump the version prior to release?

Reply:
Yeah, the behavior may be undefined here as we do not have a semantic version in the VERSION file.
What is the result of removing the action: **built_in:bump_version**. Is it going to add an entry into the change log and use the version specified in the VERSION file? 

We can bump the version on the release date if the above works for us.

The other option is to start using the semantic version 1.0.0 for the builder.